lib/gds_api/publishing_api.rb

```diff
@@ -34,6 +36,31 @@ def get_content(content_id, params = {})
     get_json(content_url(content_id, params))
   end
 
+  # Return a live content item
+  #
+  # Raises exception if the item doesn't exist.
+  # Raises an exception if the version number is passed in the params
```

Comment:
Do we need to state these if we have a raise part of this docblock?

Reply:
I'll remove the version number statement as that's no longer true.